### PR TITLE
[docs](ddl) add docs for generated column

### PR DESCRIPTION
**Pull request by feiniaofeiafei**
related code in apache/doris#35284

### CODE REVIEW

**Comment by luzhijing** [comment]
Please add En docs

**Review by shasevendow — APPROVED** [review]

**Comment by luzhijing** [comment]
Waiting for https://github.com/apache/doris/pull/35284 merged.

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-statements/Data-Definition-Statements/Create/CREATE-TABLE-AND-GENERATED-COLUMN.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+```shell
+curl --location-trusted -u root: -H "Expect:100-continue" -H "column_separator:," \
+-H "columns: a, b" -H "partial_columns:true" \
+-T /Users/moailing/Documents/tmp/gen_col_data.csv \
+http://127.0.0.1:8030/api/testdb/partial_column_unique_gen_col/_stream_load
+```
+
+
+## ALTER TABLE和生成列
+生成列暂时不支持ADD COLUMN, MODIFY COLUMN。
```
Comment (morrySnow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/sql-manual/sql-statements/Data-Definition-Statements/Create/CREATE-TABLE-AND-GENERATED-COLUMN.md:228]: 这里有点儿简略。可能会被误会为有生成列的表，都不能添加列和修改列

**Review by morrySnow — APPROVED** [review]

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**